src/components/Support/DonationInstructions.tsx

```diff
@@ -0,0 +1,38 @@
+import { cn } from "@/utils";
+import { Heading } from "../shared/Heading/Heading";
+
+interface Props {
+  includeHeader?: boolean;
+}
+
+export const DonationInstructions = ({ includeHeader }: Props) => {
+  return (
+    <>
+      {includeHeader && (
+        <Heading variant="h4" color="green" contrast="black">
+          Jak przekazać 1,5%?
+        </Heading>
+      )}
+      <div
+        className={cn(
+          "text-greenMain max-md:max-w-full desktop:text-xl",
+          !includeHeader && "mt-3",
```

Comment:
If we are not including header then we should adjust the layout based on the container (parent element).
I personally don't like to create such style dependencies. We should allow the consuming components (containers) to dictate the layout.